### DISCUSSION TITLE
Screenshot functionality

### DISCUSSION THREAD

**Discussion by Denperidge** [AprilSylph/XKit-Rewritten#1118] [Suggestion Box]
For example, it could allow posts larger than the viewport to easily be made into one solid image

Additionally - if going too far is your cup of tea - it could allow an export of a video with the rest of the post’s content

**Comment by marcustyphoon** [comment]
Interesting idea! 

Due to the APIs required to do this kind of thing, I think this would probably be its own web extension like https://github.com/AprilSylph/Outbox-for-Tumblr or https://github.com/AprilSylph/Filtering-Plus, if it were made.

That I can tell, the technique used would have to be basically the one https://github.com/mrcoles/full-page-screen-capture-chrome-extension uses, where it takes multiple screenshots and stitches them together. Might be a fun coding project if I find myself looking for one.
